packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -109,7 +127,13 @@ export class GcdsCard {
 
     this.updateLang();
 
+    this.validateCardTitle();
+    this.validateHref();
     this.validateBadge();
+
+    if (this.error.includes('href') || this.error.includes('cardTitle')) {
+      console.error('gcds-card: Error rendering, please check required properties \'card-title\' or \'href\'');
+    }
```

Comment:
Is it possible to define this as a function and call it as such `this.validateRequiredProps();` here?

And then below, call the same function again before rendering?

I think it could be possible, with multiple watches like this: https://stenciljs.com/docs/reactive-data